examples/pytorch/gat/run.sh

```diff
@@ -1,24 +1,20 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-set -ex
```

Comment:
why is it removed?

Reply:
I deleted the old gat/run.sh and replaced it with gat/geometric/run.sh

Reply:
Please, put it back.